src/electron/platform/android/setup/steps/detect-adb.ts

```diff
@@ -7,6 +7,6 @@ export const detectAdb: AndroidSetupStepConfig = deps => ({
     actions: {},
     onEnter: async () => {
         const detected = await deps.hasAdbPath();
-        deps.stepTransition(detected ? 'detect-devices' : 'prompt-locate-adb');
+        deps.stepTransition(detected ? 'prompt-connect-to-device' : 'prompt-locate-adb');
```

Comment:
Clarification of logic: Our transition to detect-devices should be independent of hasAdbPath, because detect-devices is able to use environment variables that aren't in our store. Flow should be:

1: get adbPath (null is OK)
2: Call detectDevices(adbPath)
3: If error (ADB wasn't found), prompt for ADB path. Update ADB path, then restart at step 1
4. Apply logic based on how many devices were found. 0 => prompt to connect, 1 => select it, 2+ => choose device